cmd/createtree/main_test.go

```diff
@@ -42,7 +43,12 @@ func marshalAny(p proto.Message) *any.Any {
 }
 
 func TestRun(t *testing.T) {
-	pemPath, pemPassword := "../../testdata/log-rpc-server.privkey.pem", "towel"
+	err := os.Chdir("../..")
```

Comment:
Strictly, you can't rely on the working directory being the directory that the test is in. However, since this test already did that, I'm ok with not fixing this issue. If you want to fix it, you could use the `testonly.RelativeToPackage()` func to get the absolute path of these files.